### PR TITLE
Update README to reference Doxx project

### PR DESCRIPTION
**Pull request by kb2ma**
The project README includes a _Using Templates_ section at the bottom that references dynamically generated pages (DGPs) and shows an example. Recently, I wanted to learn more about DGPs, @vipulgupta2048 referred me to the Doxx documentation. So, for future learners this PR adds a reference to the Doxx project README, which contains useful information about DGPs.

The PR also updates the example use of a template to select a DGP, which contained outdated variable names.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,12 +128,12 @@ In dynamically generated pages (the ones with the device-type and language dropd
...
 
 ```
-Getting started with {{ $device_details.name }} using {{ $language_details.name }}  
+Getting started with {{ $device.name }} using {{ $language.name }}
 
 Select the {{ $device.name }} device type`
 ```
 
-To add additional dynamic properties, see the files in `/config/dictionaries`.
+To add additional dynamic properties, see the files in `/config/dictionaries`. To learn more about dynamically generated pages, see the [README](https://github.com/balena-io/doxx) for the Doxx project, on which the balenaCloud documentation is built.
```
Comment (vipulgupta2048) [inline README.md:136]:
```suggestion
To add additional dynamic properties, see the files in `/config/dictionaries`. To learn more about dynamically generated pages, see the [README](https://github.com/balena-io/doxx) for the Doxx project using which the balenaCloud dynamic documentation is generated.
```

Good to refer this, LGTM. Ship it.

Comment (kb2ma) [reply README.md]: Thanks. Tweaked your suggestion a little.

**Comment by kb2ma** [comment]
@balena-ci retest

**Comment by kb2ma** [comment]
@balena-ci I self-certify!

**Pull request merged**